### DISCUSSION TITLE
Issue instantiating secureLocalStorage in types.d.ts file

### DISCUSSION THREAD

**Discussion by jordanwmckee** [sushinpv/react-secure-storage#23] [General]
Not sure if this could be an error on my end or something, as I am new to Vite/React and ts, but I am getting this error with the `types.d.ts` file that is preventing me from building my app to test it out.

<img width="1257" alt="image" src="https://user-images.githubusercontent.com/77068712/234401653-6be4a461-5b5b-42c2-94de-1c3d9ec72d80.png">

**Comment by sushinpv** [comment]
@jordanwmckee Is it possible to create a repository and upload your sample code with this issue ? I'm not able to reproduce in React and Next.js with latest version of Typescript with strict mode

**Reply by jordanwmckee** [reply]
I created a repo to reproduce the issue, but also was not getting it this time; so I went through and updated my tsconfig and package.json to the latest versions and that seemed to solve the issue. Sorry for the confusion!
